src/AzureDeprecation.APIs.REST/DataAccess/Models/PaginationNoticesRequest.cs

```diff
@@ -0,0 +1,15 @@
+using System.ComponentModel.DataAnnotations;
+
+namespace AzureDeprecation.APIs.REST.DataAccess.Models;
+
+public class PaginationNoticesRequest
+{
+    public const int DefaultPageSize = 100;
```

Comment:
Let's use smaller page size
```suggestion
    public const int DefaultPageSize = 25;
```

Reply:
Yeah, make sense